module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -24,12 +24,16 @@ Read the Base `R` documentation [**here**](https://rdrr.io/r/).
 | Base `R`| [`<-function(x) { <code> }`](https://adv-r.hadley.nz/functions.html) | Function | Creates a function that would take the defined parameters as input and execute the commands within the curly braces  |
```

Comment:
I realized that we don't have a link to the `\()` anywhere... I don't know if this is the right place, but we should put it somewhere... Maybe with `purrr`? Docs here: https://rdrr.io/r/base/function.html